apps/prairielearn/src/server.js

```diff
@@ -1290,6 +1290,21 @@ export async function initExpress() {
     },
     (await import('./pages/instructorQuestions/instructorQuestions.js')).default,
   ]);
+  app.use('/pl/course_instance/:course_instance_id(\\d+)/instructor/ai_generate_question_jobs', [
+    function (req, res, next) {
+      next();
+    },
```

Comment:
Presumably this is left over from a previous iteration? Please remove this and other unnecessary middlewares.

Reply:
Done in deabe2646069d6cce9d383460ea361768ab6c66c